omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -352,6 +352,14 @@
         "url": "reportingui/runReport.page?reportDefinition=fe02c34a-5c67-453b-93e9-4946e7bd8c59",
         "order": 35
       },
+      {
+        "id": "HMIS 105  4",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.monthly",
+        "type": "link",
+        "label": "HMIS 105 SECTION 4:  HCT",
```

Comment:
@solemabrothers change label to "HMIS 105 Section 4: HTS" which is the title of the report pages

Reply:
OK